.gitignore

```diff
@@ -10,4 +10,3 @@ coverage
 .idea
 *.iml
 .eslintcache
-
```

Comment:
I have configured my editor to add an newline to the end of each file. 

https://www.google.ch/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=file+ends+with+empty+line